cmd/virt-launcher/virt-launcher.go

```diff
@@ -43,6 +43,10 @@ func markReady(readinessFile string) {
 }
 
 func createSocket(socketDir string, namespace string, name string) net.Listener {
+	socketPath := isolation.SocketFromNamespaceName(socketDir, namespace, name)
+	if err := os.RemoveAll(socketPath); err != nil {
```

Comment:
don't you just need os.Remove() instead of RemoveAll? In this situation, both work fine, but they mean different things.

Reply:
os.Remove() returns an error if the file/path does not exist, os.RemoveAll() returns nil in that case: https://play.golang.org/p/5tOXyB3T4b